rust_src/src/eval.rs

```diff
@@ -37,6 +41,58 @@ use crate::{
  *   and temporaries from garbage collection while it needs them.      *
  * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 
+#[no_mangle]
```

Comment:
Could you break these out into a separate commit? It seems like a significant change that's worth recording apart from `call-process-region`.

Reply:
Sure thing.

Reply:
These changes are now in #1248.